hedera-node/src/main/java/com/hedera/services/store/contracts/HederaWorldState.java

```diff
@@ -109,57 +111,60 @@ public HederaWorldState(
 		sponsorMap.forEach((contract, sponsorAddress) -> {
 			if (!provisionalAccountUpdates.containsKey(contract)) return;
 
-			AccountID sponsorAccount = EntityIdUtils.accountParsedFromSolidityAddress(sponsorAddress.toArrayUnsafe());
+			AccountID sponsorAccount = accountParsedFromSolidityAddress(sponsorAddress.toArrayUnsafe());
 
 			WorldStateAccount provisionalAccount = provisionalAccountUpdates.get(contract);
 			if (ledger.exists(sponsorAccount)) {
 				MerkleAccount sponsorMerkleAccount = ledger.get(sponsorAccount);
 				provisionalAccount.setProxyAccount(sponsorMerkleAccount.getProxy());
```

Comment:
Are we not setting the key and memo of `CREATE`ed accounts then? (I'm not fully recalling the discussion on the call)